pages/api/settings/collect.js

```diff
@@ -0,0 +1,25 @@
+import prisma from '@prisma/index'; // Prisma client
+import { getSession } from 'next-auth/client'; // Session handling
+
+// get settings by specifying the corresponding type
+// POST with { type : type }
+export default async (req, res) => {
+  // Collect session from request
+  const session = await getSession({ req });
+
+  const { type: type } = req.body;
```

Comment:
No need to rename when destructuring object.
```suggestion
  const { type } = req.body;
```